examples/filter/pipeline_passthru/README.md

```diff
@@ -0,0 +1,8 @@
+# Audio passthru
+
+This demo passes audio received at the aux_in port to the headphone and speaker outputs.
+
+To run this example you need ESP32 LyraT or compatible board:
```

Comment:
Could you add here couple of good points from the PR summary? 

* Useful to exercise the audio pipeline from end to end when bringing up new hardware.
* Typically used for production line sanity checking and performance checking of the audio signal path by taking THD+N measurements using an Audio test set.
* Useful when checking for left and right channel consistency through audio path.